neqo-http3/src/connection.rs

```diff
@@ -187,6 +187,19 @@ impl ::std::fmt::Display for Http3Connection {
     }
 }
 
+macro_rules! transaction {
+    ($t:expr) => {
+        match $t {
+            Some(t) => t,
+            _ => {
+                return Err(Error::TransportError(
+                    neqo_transport::Error::InvalidStreamId,
```

Comment:
If this code is generating the error, does it really make sense to use a TransportError?

Reply:
having a transaction should match having a quic stream, therefore I use this error. I could make a new one.